Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Templates/ITemplatePart.cs

```diff
@@ -35,7 +35,7 @@ public interface ITemplatePart
     public static class Extensions
     {
         /// <summary>
-        /// Attempts to replace the token if it's available and returns true, otherwise returns false.
+        /// Attempts to replace the token if its available and returns true, otherwise returns false.
```

Comment:
A few more of those comment regressions (I think they were all changes taken from the previous PRs as suggestions within the PR)

```suggestion
        /// Attempts to replace the token if it's available and returns true, otherwise returns false.
```